src/main/java/org/folio/consortia/domain/dao/FolioAuditorAware.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.consortia.domain.dao;
```

Comment:
This package should not be located in domain, please move it to config package

Reply:
fixed